reconcile/utils/terraform_client.py

```diff
@@ -228,22 +228,22 @@ def log_plan_diff(
                 if action == "no-op":
                     logging.debug([action, name, resource_type, resource_name])
                     continue
-                # Ignore RDS modifications that are going to occur during the next
-                # maintenance window. This can be up to 7 days away and will cause
-                # unnecessary Terraform state updates until they complete.
-                if (
-                    action == "update"
-                    and resource_type == "aws_db_instance"
-                    and self._is_ignored_rds_modification(
-                        name, resource_name, resource_change
-                    )
-                ):
-                    logging.debug(
-                        f"Not setting should_apply for {resource_name} because the "
-                        f"only change is EngineVersion and that setting is in "
-                        f"PendingModifiedValues"
-                    )
-                    continue
+                if action == "update":
+                    if resource_type == "aws_db_instance":
+                        pending_rds_modifications = self._get_pending_rds_modification(
+                            name, resource_name, resource_change
+                        )
+                        # Ignore RDS modifications that are going to occur during the next
+                        # maintenance window. This can be up to 7 days away and will cause
+                        # unnecessary Terraform state updates until they complete.
+                        if pending_rds_modifications:
+                            logging.warning(
```

Comment:
The reconcile channel would show this on each loop unless we either ignore it, or change this to debug. I'm fine with either because the warning could be useful for debugging in the pod logs.

Reply:
@steveteahan, ah yes, doh! I forgot that we stream logs into the channel - for some reason I thought that these logs are going into the container exclusively, and then we push messages to Slack. Sorry! I will drop this change.

Reply:
Done. Thank you!